xbmc/video/windows/GUIWindowFullScreen.cpp

```diff
@@ -104,6 +105,18 @@ CGUIWindowFullScreen::~CGUIWindowFullScreen(void)
 
 bool CGUIWindowFullScreen::OnAction(const CAction &action)
 {
+  if (CSeekHandler::GetInstance().HasTimeCode() && action.GetButtonCode())
```

Comment:
don't care about that IMO. The reason for it is to have a different button mapped to select. IMO we should load the default instead of that virtual window.